requirements/broken-projects.in

```diff
@@ -0,0 +1,2 @@
+Pillow < 10  # https://github.com/blockdiag/sphinxcontrib-blockdiag/issues/26
+pydantic < 2  # looks like `python-on-whales` hasn't upgraded to support v2
```

Comment:
Do we want to track these? Normally easier to just have the locks at working versions and let Dependabot PRs figure out when they are passing and can be bumped.

Reply:
Yes, otherwise pip-tools will try to upgrade to non-working versions. And dependabot would also spam with PRs likely.

FWIW, we may want to reconsider after getting the hotfixes and backports for this in.

Reply:
If it's PRs to bump pillow and pydantic, that'd be the expected behaviour to me. Once they start working, the PRs will get merged, and I can check open PRs to see which are still failing at any given time.